### PR TITLE
support quantized model for generation task in chat

### PR DESCRIPTION
**Pull request by PenghuiCheng**
## Type of Change

feature
No API changed

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

**Review by hshen14 — COMMENTED** [review]
Please add note in README and dockerfile for MPT due to IPEX issue.

File: workflows/chatbot/inference/generate.py [hunk 1]
```diff
@@ -559,7 +605,7 @@ def predict_stream(**params):
     max_new_tokens = (
         int(params["max_new_tokens"]) if "max_new_tokens" in params else 256
     )
-    do_sample = params["do_sample"] if "do_sample" in params else True
+    do_sample = False #params["do_sample"] if "do_sample" in params else True
```
Comment (hshen14) [inline workflows/chatbot/inference/generate.py:608]: Please revert the hard code

File: workflows/chatbot/inference/generate.py [hunk 2]
```diff
@@ -571,7 +617,7 @@ def predict_stream(**params):
     force_words_ids = params["force_words_ids"] if "force_words_ids" in params else None
     use_hpu_graphs = params["use_hpu_graphs"] if "use_hpu_graphs" in params else False
     use_cache = params["use_cache"] if "use_cache" in params else True
-    return_stats = params["return_stats"] if "return_stats" in params else False
+    return_stats = True #params["return_stats"] if "return_stats" in params else False
```
Comment (hshen14) [inline workflows/chatbot/inference/generate.py:620]: same as above

File: workflows/chatbot/inference/generate.py [hunk 3]
```diff
@@ -611,14 +657,14 @@ def predict_stream(**params):
         stop_token_ids.append(end_token_id)
         generation_config = GenerationConfig(
             temperature=temperature,
-            top_p=top_p,
-            top_k=top_k,
-            repetition_penalty=repetition_penalty,
+            #top_p=top_p,
```
Comment (hshen14) [inline workflows/chatbot/inference/generate.py:660]: same

**Comment by kevinintel** [comment]
please fix conflict

**Pull request closed without merging**